src/athena.js

```diff
@@ -63,21 +63,25 @@ export let queryResultToObjectsArray = function(queryResult) {
     _.forEach(columns, function(column, columnIndex) {
       let value = _.get(row, `Data[${columnIndex}].VarCharValue`);
 
-      if (_.isDefined(value)) {
-        switch (_.toLower(column.Type)) {
-        case ('integer'):
-        case ('tinyint'):
-        case ('smallint'):
-        case ('bigint'):
-        case ('double'):
-          value = Number(value);
-          break;
-        case ('boolean'):
-          value = Boolean(value);
-          break;
-        default:
-          break;
+      try {
+        if (_.isDefined(value)) {
+          switch (_.toLower(column.Type)) {
+          case ('integer'):
+          case ('tinyint'):
+          case ('smallint'):
+          case ('bigint'):
+          case ('double'):
+            value = Number(value);
+            break;
+          case ('boolean'):
+            value = Boolean(value);
+            break;
+          default:
+            break;
+          }
```

Comment:
why try catch?
not enough to use _.toNumber ?

granted I don't know what you get when 'boolean' . there's no _.toBoolean https://github.com/lodash/lodash/pull/2066

Reply:
The reason I added this is primarily for column names, which will always be strings in the first row (while column values could be something else, but we haven't had such examples on practice). To fix it more properly, I would have to drop column names row before passing it into this function and then it would require change of the API, because right now `queryResultToObjectsArray` is not the best with accepting queryResult, while it's better to accept `queryResult.ResultSet` (which I would like to do, but separately, I guess). 

That's why I decided to make a safe fallback to strings.